### ISSUE TITLE
instance_loss_k & instance_loss_m

### ISSUE THREAD

**Issue by Scott-che** [uncbiag/SimpleClick#17]
I would like to ask the training loss. It is normal that instance_loss_k and instance_loss_m increase when training.
![image](https://github.com/uncbiag/SimpleClick/assets/67545227/fc669f73-7472-4e29-a42a-fd33fdad4380)

**Comment by Scott-che** [comment]
Is you instance loss keep increasing when training ?
![image](https://github.com/uncbiag/SimpleClick/assets/67545227/a2798d62-0d4b-45db-9164-b1b2528feb22)
I just modify batch size to 8.

**Comment by Scott-che** [comment]
Though the instance loss keeps increasing, iou keep increasing.
Is it normal?

**Comment by qinliuliuqin** [comment]
Actually, I didn't monitor these intermediate metrics as they are not involved in the loss backpropagation. I may take some time to read the code details as they are inherited from RITM. How about your testing performance?

**Comment by Scott-che** [comment]
I trained the model, sbd_plainvit_base448, on single 2080ti, and just modified the batch size to 8.
Train 55 epochs.
Eval results for model: last_checkpoint
-----------------------------------------------------------------------------------------------
|  BRS Type   |  Dataset  | NoC@80% | NoC@85% | NoC@90% |>=20@85% |>=20@90% | SPC,s |  Time   |
-----------------------------------------------------------------------------------------------
|    NoBRS    |    SBD    |  2.50   |  3.34   |  5.30   |   243   |   699   | 0.058 | 0:34:07 |

**Comment by qinliuliuqin** [comment]
It's comparable to my results as shown in Tab. 2 of the paper. My batch size for ViT-B is 140 and I used 4 A6000 GPUs. Since your batch size is 8, so your training iterations are 140/8 times higher than mine.

**Comment by Scott-che** [comment]
In your comment, I need to train more epochs than you.

**Comment by qinliuliuqin** [comment]
No, I didn't mean that. I meant your training iterations were much higher but got slightly worse results than mine. We all trained the ViT-B model with 55 epochs but used different training batches (140 vs. 8). Therefore, we have different training iterations (N*55/140 vs. N*55/8, N is the number of training images).

**Comment by Scott-che** [comment]
Is it possible to get the similar performance that shows on the paper with training on single 2080ti?

**Comment by qinliuliuqin** [comment]
Your results are very close to mine now given the differences in our training environments.
